File: test/fixtures/python/basic.py
from decimal import Decimal


def calculate_total(qty, price):
    return qty * price


format_price = lambda price: f"${price:.2f}"


class InvoiceService:
    def save(self):
        return True

    @staticmethod
    def build():
        return InvoiceService()


class ReceiptService:
    def save(self):
        return False

File: test/fixtures/typescript/basic.ts
import { Invoice } from "./invoice";

export interface LineItem {
  qty: number;
  price: number;
}

export type Total = number;

export function calculateTotal(qty: number, price: number): Total {
  return qty * price;
}

export const formatPrice = (price: number): string => {
  return `$${price.toFixed(2)}`;
};

class InvoiceService {
  save(invoice: Invoice): boolean {
    return true;
  }
}

class ReceiptService {
  save(): boolean {
    return false;
  }
}
